### PR TITLE
DDCE-3463 - Fix error message for exclude nino, + deps upgrade, + cle…

### PR DESCRIPTION
**Pull request by frikit**
…anup, + increase coverage

# DDCE-3463

Please include a summary / description of the change and which issue it fixes.  Include any relevant user needs, context or links to other PRs related to this PR (eg. acceptance tests, environment config).

## Checklist PR Raiser
- [x]  I've updated the README if there are functionality changes
- [x]  I've executed the project script to check Dependencies and update them if they're non-breaking changes
- [x]  I've executed Scalastyle and fixed any warnings that can be resolved quickly
- [x]  I've executed the acceptance test pack locally to ensure there are no functional regressions (Performance tests where relevant)
- [x]  I've included appropriate tests with any code I've added (Unit, Integration), and it meets Coverage
- [x]  I've squashed my commits - including the JIRA issue number in the commit message
- [x]  I've contacted a BA to create a task if any of these actions would result in a larger piece of work

### CODE REVIEW

File: test/utils/SystemPropertiesHelperSpec.scala [hunk 1]
```diff
@@ -74,7 +74,7 @@ class SystemPropertiesHelperSpec extends AnyWordSpecLike with Matchers with Opti
   "When getting a String system property which does exist the helper" should {
     "return the default value" in {
       val s = new StubSystemProperties
-      assert(s.getStringProperty("searchString", null) == "foundString")
```
Comment (lmugnai) [inline test/utils/SystemPropertiesHelperSpec.scala:77]: getStringProperty is not being used. Can we remove it?

File: test/utils/SystemPropertiesHelperSpec.scala [hunk 2]
```diff
@@ -60,7 +60,7 @@ class SystemPropertiesHelperSpec extends AnyWordSpecLike with Matchers with Opti
   "When getting a Boolean system property which does exist the helper" should {
     "return the default value" in {
       val s = new StubSystemProperties
-      assert(s.getBooleanProperty("searchBoolean", false) == true)
```
Comment (lmugnai) [inline test/utils/SystemPropertiesHelperSpec.scala:63]: getBooleanProperty is not being used. Can we remove it?

File: test/utils/SystemPropertiesHelperSpec.scala (hunk 1 above)
Comment (frikit) [reply test/utils/SystemPropertiesHelperSpec.scala:77]: done

File: test/utils/SystemPropertiesHelperSpec.scala (hunk 2 above)
Comment (frikit) [reply test/utils/SystemPropertiesHelperSpec.scala:63]: done

**Review by lmugnai — APPROVED** [review]

**Pull request merged**